recipe/0103-poetry-reading-annotations/manifest2.json

```diff
@@ -0,0 +1,58 @@
+{
+   "@context":"http://iiif.io/api/presentation/3/context.json",
+   "id":"{{ id.url }}",
+   "type":"Manifest",
+   "label":{
+      "en":[
+         "Anne Sexton, Poetry Reading, 1974 -- Her Kind"
+      ]
+   },
+   "items":[
+      {
+         "id":"{{ id.path }}/canvas/1",
+         "type":"Canvas",
+         "duration":107,
+         "items":[
+            {
+               "id":"{{ id.path }}/canvas/1/paintings",
+               "type":"AnnotationPage",
+               "items":[
+                  {
+                     "id":"{{ id.path }}/canvas/1/painting/1",
+                     "type":"Annotation",
+                     "motivation":"painting",
+                     "body":{
+                        "id":"https://library.harvard.edu/poetry/audio/listeningbooth/PS3537E915A6x1974/Her_Kind.mp3",
+                        "type":"Sound",
+                        "format":"audio/mp3",
+                        "duration":107
+                     },
+                     "target":"{{ id.path }}/canvas/1"
+                  }
+               ]
+            }
+         ]
+      }
+   ],
+   "annotations":[
```

Comment:
This 'annotations' field also I think should be part of the canvas not part of the manifest...

Reply:
Now fixed.